docs/source/support/release-notes.rst

```diff
@@ -127,6 +133,8 @@ Sandbox
 
 - Introduced a new API for party management.
   See `#1312 <https://github.com/digital-asset/daml/issues/1312>`__.
+- Introduced a new API for package management.
```

Comment:
This has followed its context into an existing release; needs to move back to HEAD before merge.

Reply:
Thanks for the reminder, rebasing onto master usually introduces errors in the release notes and I forgot to fix them.